data/src/main/java/org/oppia/data/backends/gae/model/SubtopicSummary.kt

```diff
@@ -0,0 +1,13 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
+data class SubtopicSummary(
+
+  @Json(name = "id") val id: String?,
```

Comment:
Perhaps ``subtopicId`` might be clearer?

Reply:
Changed id to subtopicId